core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationSummaryInfo.scala

```diff
@@ -136,4 +147,18 @@ class SingleAppSummaryInfoProvider(val app: ApplicationSummaryInfo)
   override def getRapidsJars: Seq[String] = {
     app.rapidsJar.map(_.jar).seq
   }
+
+  override def getDistinctLocationPct: Double = {
+    100.0 * distinctLocations.size / app.dsInfo.size
+  }
+
+  override def getRedundantReadSize: Long = {
+    distinctLocations
+      .filter {
+        case(_, objects) => objects.size > 1 && objects.exists(_.format.contains("Parquet"))
```

Comment:
Nit:

```suggestion
        case(_, objects) => objects.size > 1 && objects.exists(isFormatSupportedInFileCache(_))
```

And we can define the following in the class to allow for future extension

```
// later we can add the ORC when it is ready
private val fileCacheSupportedFormats = Set("Parquet")

def isSupportedForFileCache(locObj: DataSourceProfileResult) : boolean = {
  fileCacheSupportedFormats.contains(locObj.format)
}
```

Pros: easy to add new formats in future; and allow to add `isSupportedForFileCache` in unit tests

Reply:
Yep, that would be good.  Added that idea in the future issue: https://github.com/NVIDIA/spark-rapids-tools/issues/362.